python/ray/train/torch/xla/config.py

```diff
@@ -0,0 +1,127 @@
+
+import os
+import uuid
+from dataclasses import dataclass
+
+import ray
+from ray._private.ray_constants import NEURON_CORES
+from ray.train._internal.utils import get_address_and_port
+from ray.train._internal.worker_group import WorkerGroup
+from ray.train.backend import Backend
+from ray.train.torch import TorchConfig
+from ray.util import PublicAPI
+
+
+@PublicAPI(stability="alpha")
+@dataclass
+class TorchXLAConfig(TorchConfig):
+    """
+    Configuration for torch XLA setup.
+    See https://pytorch.org/xla/release/1.13/index.html for more info.
+
+    Args:
+        runtime: Runtime to use for training. Supported values are "xrt", "pjrt".
+            Currently, only "xrt" is supported.
+        accelerator_type: The accelerator type used to differentiate the XLA backend.
+            Currently, only "neuron_cores" is supported.
+
+    """
+
+    runtime: str = "xrt"
+    accelerator_type: str = NEURON_CORES
+
+    @property
+    def backend_cls(self):
+        if self.accelerator_type == NEURON_CORES:
+            return _TorchAwsNeuronXLABackend
+        else:
+            raise NotImplementedError
```

Comment:
if other accelerator types are not supported, do we need to make `accelerator_type` configurable at all?

Reply:
It can extendible for TPU (similar class of xla).

Reply:
Can we just `runtime` and `accelerator_type` attributes in this PR to keep the surface area simple? We can document in the class docstring that only this combination is supported right now.

Once we want to add  support TPU and/or pjrt it will be better to make sure the interface and testing is comprehensive then.

Reply:
@matthewdeng To confirm, we want to remove runtime/acclerator_type and move the information to docs?

Reply:
Yeah more specifically can just be in the `TorchXLAConfig` docstring - I think this would be the simplest and cleanest solution fro now.